package/Editor/IOSPostBuildProcessor.cs

```diff
@@ -0,0 +1,26 @@
+﻿#if UNITY_EDITOR
+
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+
+public class IOSPostBuildProcessor
+{
+    [PostProcessBuild]
+    public static void OnPostprocessBuild(BuildTarget target, string path) {
+        if (target == BuildTarget.iOS) {
+            string projPath = PBXProject.GetPBXProjectPath(path);
+
+            PBXProject proj = new PBXProject();
+            proj.ReadFromFile(projPath);
+
+            string targetGUID = proj.TargetGuidByName("Unity-iPhone");
+            proj.AddBuildProperty(targetGUID, "SWIFT_VERSION", "5.0");
```

Comment:
This can be addressed later but we should be sure that the version of Swift that is used does not conflict with the project that is generated by Unity across different unity versions. I don't know what that looks like and I can't see any documentation about it but just something to think about.

Reply:
Yeah, let's see how it will impact pubs, and then we can address it if it becomes an issue.